pkg/boot/ebda/ebda_test.go

```diff
@@ -43,6 +43,11 @@ func fakeEBDA(sizeKB int) []byte {
 	return b
 }
 
+// currently the only offset passed to fakeDevMemEBDA is 0x9000
+// hardcoding seems not be a good idea since we may need this function for
+// future testing
+//
+//nolint:unparam
```

Comment:
Again, the common way to indicate an unused parameter is
func f(_ int, something int)
does golangci-lint not understand this? 
I really don't like these no lint: things everywhere. It means, when code changes, people also need to understand to change the nolint, and they will forget. And then we'll have a bogus no lint directive. It's why these extraterritorial things are such a problem. 
Does golangci-lint understand _ parameters? If not, it should.

Reply:
Just confirmed: _ the param and golangci-lint will not complain.
remove this nolint: there is a Go-standard way to do this.